scene/3d/skeleton_ik_3d.cpp

```diff
@@ -523,6 +524,10 @@ void SkeletonIK3D::set_max_iterations(int p_iterations) {
 	max_iterations = p_iterations;
 }
 
+Skeleton3D *SkeletonIK3D::get_parent_skeleton() const {
+	return cast_to<Skeleton3D>((Object *)skeleton_ref);
```

Comment:
This doesn't catch cases when the skeleton is freed, I think you need
```suggestion
	return cast_to<Skeleton3D>(skeleton_ref.get_validated_object());
```

Reply:
Wow, I had assumed that `Variant::operator Object*` validates the ObjectID. good to know. Changed to get_validated_object.

Reply:
It avoids doing so for performance